### ISSUE TITLE
Clarify that path for extracting OpenRefine should be 'shallow'

### ISSUE THREAD

**Issue by bencomp** [datacarpentry/openrefine-socialsci#109]
In the workshop today, several people got an error about "Pathname too long" when they extracted the ZIP file that had OpenRefine. We should make clear that the location where they extract the ZIP file should not be 'deep' in the file system, i.e. not `G:\Documents\Projects\something something\Year 2\Data Carpentry workshop\Day 1`, to prevent this error.

I found that suggesting to extract to the desktop worked, even though it's not great as a permanent location.

**Comment by CosiMichele** [comment]
Hello, I found this issue through the [carpentries' help wanted page](https://carpentries.org/help-wanted-issues/) (doing my checkout), so I hope it is okay to post/comment.

Firstly, I'd suggest that for any workshop all paths remain short and/or close to Home. I've noticed that this helps the students in two ways: 1. it makes it easier for them to return to the same folder (after a break for example, or if the workshop is multiple days long - it isn't uncommon to "get lost" in your own system of directories), 2. it prevents these sort of issues.

That said, this isn't exactly an OpenRefine issue, but something related to Windows and Windows Registry. As it was pointed out in https://github.com/OpenRefine/OpenRefine/issues/2217, the issue is related to the maximum amount of characters that  Windows recognizes when typing in paths. Enabling `LongPathsEnabled` in the Windows Registry will allow much longer paths to be used.

I hope this helps with your issue and future workshops!
